openedx/core/lib/xblock_utils.py

```diff
@@ -396,3 +397,29 @@ def _course_info_content(html_parsed):
                         return payload
 
     return course_update_items
+
+
+def add_inline_analytics(user, block, view, frag, context):  # pylint: disable=unused-argument
```

Comment:
We should move this function directly `openedx.core.lib.inline_analytics_utils`.
It removes an additional import, while also leaving this file here clean of any diff (at least for this).